webapp/src/main/java/com/box/l10n/mojito/smartling/SmartlingClientConfiguration.java

```diff
@@ -62,14 +60,10 @@ public OAuth2RestTemplate smartlingRestTemplate() {
         RestTemplateUtils restTemplateUtils = new RestTemplateUtils();
         restTemplateUtils.enableFeature(oAuth2RestTemplate, DeserializationFeature.UNWRAP_ROOT_VALUE);
 
-        AccessTokenProviderChain accessTokenProviderChain = new AccessTokenProviderChain(Arrays.asList(
-                new SmartlingAuthorizationCodeAccessTokenProvider())
-        );
-        oAuth2RestTemplate.setAccessTokenProvider(accessTokenProviderChain);
-
-        DefaultUriTemplateHandler defaultUriTemplateHandler = new DefaultUriTemplateHandler();
-        defaultUriTemplateHandler.setBaseUrl(baseUri);
+        oAuth2RestTemplate.setAccessTokenProvider(new SmartlingAuthorizationCodeAccessTokenProvider());
+        oAuth2RestTemplate.setRetryBadAccessTokens(true);
 
+        DefaultUriBuilderFactory defaultUriTemplateHandler = new DefaultUriBuilderFactory(baseUri);
```

Comment:
These are drop-in replacements for classes that got deprecated during the Spring boot 2 migration